app/views/journeys/expenses/advertisingOrMarketing/AdvertisingAmountView.scala.html

```diff
@@ -14,17 +14,18 @@
  * limitations under the License.
  *@
 
-@import models.common.{BusinessId, TaxYear, UserType}
-@import viewmodels.InputWidth._
 @import controllers.journeys.expenses.advertisingOrMarketing.routes
+@import models.common.{BusinessId, TaxYear, UserType}
 @import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
+@import viewmodels.InputWidth._
+@import views.html.components._
```

Comment:
All are wildcard imports because I will be making other PRs replacing view components which I will also be importing from `views.html.compontents`